content/src/service/synchronization/EventDeployer.ts

```diff
@@ -35,13 +36,16 @@ export class EventDeployer {
     EventDeployer.LOGGER.trace(`Downloading files for entity (${deployment.entityType}, ${deployment.entityId})`)
 
     // Download the entity file
+    const stopTimer = DCL_CONTENT_DOWNLOAD_TIME.startTimer({ remote_catalyst: source?.getAddress() || 'undefined' })
     const entityFile: Buffer | undefined = await this.getEntityFile(deployment, source)
+    stopTimer()
 
     const { auditInfo } = deployment
 
     if (entityFile) {
       const isLegacyEntity = !!auditInfo.migrationData
       if (auditInfo.overwrittenBy) {
+        DCL_CONTENT_DOWNLOADED_TOTAL.inc({ overwritten: 'true' })
```

Comment:
what does this do?

Reply:
It increments DCL_CONTENT_DOWNLOADED_TOTAL with the tag overwritten: true, the intended usage is to start measuring how many times this branch is taken. and to measure the difference between `DCL_CONTENT_DOWNLOAD_TIME.count <> sum(DCL_CONTENT_DOWNLOADED_TOTAL)` to detect how many entitiyFile are undefined for some reason